src/cf.data.pre

```diff
@@ -1068,7 +1068,7 @@ DEFAULT: ssl::certUntrusted ssl_error X509_V_ERR_INVALID_CA X509_V_ERR_SELF_SIGN
 DEFAULT: ssl::certSelfSigned ssl_error X509_V_ERR_DEPTH_ZERO_SELF_SIGNED_CERT
 ENDIF
 DEFAULT: all src all
-DEFAULT: manager url_regex -i ^cache_object:// +i ^[^:]+://[^/]+/squid-internal-mgr/
+DEFAULT: manager url_regex +i ^[^:]+://[^/]+/squid-internal-mgr/
```

Comment:
Now that the scheme does not matter we can make this a faster path-only check.
```suggestion
DEFAULT: manager urlpath_regex +i ^/squid-internal-mgr/
```

Reply:
Please do _not_ change the type of the `manager` ACL in this PR, especially for optimization reasons. Doing so will break existing Squid configurations that explicitly adjust the `manager` ACL. Such configurations are problematic, and we are working on prohibiting such adjustments, but that ban should be done (differently) in a separate PR (which is nearly ready for the official review).

Reply:
> a separate PR (which is nearly ready for the official review)

Yes, it would be better to avoid these changes, since that PR should cover this area.

Reply:
Is this other PR ready or in yet?
I would definitely like to reduce the security problems the current manager ACL definition could lead to.

Reply:
This PR is passing or internal review - it is not official yet.

Reply:
Please note that the `manager` ACL type should not be changed in _this_ PR regardless of that type (doomed) future. 

> I would definitely like to reduce the security problems the current manager ACL definition could lead to.

Same here, but neither the current type, nor the type suggested in this change request can solve those problems. Please resolve this change request. Hopefully, we will soon post an official PR that properly solves those problems, once and for all, but even if we fail to post that "proper fix" PR soon, any type change for this ACL is outside this PR scope and needs a dedicated PR (for the reasons already [given](https://github.com/squid-cache/squid/pull/1250#discussion_r1093314706)).